### PR TITLE
Fixes from Markdown and Yaml linters + spelling typos

### PR DESCRIPTION
**Pull request by karianna**
##### Checklist

- [ ] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] FAQ.md updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] inventory changes, ensure bastillion is updated accordingly

### CODE REVIEW

File: .github/workflows/build_vagrant.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
         sed -i -e "s/.*hosts:.*/- hosts: all/g" playbooks/AdoptOpenJDK_Unix_Playbook/main.yml
         awk '{print}/^\[defaults\]$/{print "private_key_file = id_rsa"; print "timeout = 30"; print "remote_tmp = $HOME/.ansible/tmp"}' < ansible.cfg > ansible.cfg.tmp && mv ansible.cfg.tmp ansible.cfg
 
-    - name: Run Asible Playbook
+    - name: Run Ansible Playbook
```
Comment (sxa) [inline .github/workflows/build_vagrant.yml:38]: Thanks - I was going to fix this one but you've beat me to it :-)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,27 +75,26 @@ Fixes: #1234
...
+
 1. Fork this repository
 1. Create a branch off your fork
 1. Make the change
 1. Test it (see below)
 1. Submit a Pull Request
 
-Only reviewers in the [admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure) team have permission to merge requests for this `openjdk-infrastructure` repo, 
-so please ask one of those team members to review your Pull Request. 
+Only reviewers in the [admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure) team have permission to merge requests for this `openjdk-infrastructure` repo, so please ask one of those team members to review your Pull Request.
```
Comment (sxa) [inline CONTRIBUTING.md:85]: This should now be https://github.com/orgs/AdoptOpenJDK/teams/infrastructure

File: ONBOARDING.md [hunk 3]
```diff
@@ -6,17 +6,17 @@
...
 - Ideally request users public GPG key as well as their public SSH key.
 
 ## GitHub
 
 Add the user to the correct Infrastructure team:
 
-* [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure) - Super Users
-* [@infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure) - Core Infra Team
-* [@adoptopenjdk-infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/adoptopenjdk-infrastructure) - Can be assigned Infra Issues
-* [@jenkins-admins](https://github.com/orgs/AdoptOpenJDK/teams/jenkins-admins) - Super users on Jenkins
```
Comment (sxa) [inline ONBOARDING.md:19]: This section likely needs a bit of an update now that https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1084 is complete, but I'm happy for that to be left for a separate PR

**Review by sxa — APPROVED** [review]
Minor issues relating to the group names being out of date, but I'll approve regardless as they can be resolved in other PRs if you don't want to do it here. Thanks for the changes, although is there a specific reason why you switched all the URLs to be around `[URL](url)` instead of just having them raw in the file since github renders URLs in the same way I believe (and it avoids duplication of the address)

**Comment by karianna** [comment]
> Minor issues relating to the group names being out of date, but I'll approve regardless as they can be resolved in other PRs if you don't want to do it here. Thanks for the changes, although is there a specific reason why you switched all the URLs to be around `[URL](url)` instead of just having them raw in the file since GitHub renders URLs in the same way I believe (and it avoids duplication of the address)

Markdown Linter told me off :-)

**Comment by sxa** [comment]
> Markdown Linter told me off :-)

Harsh! :-)
